internal/app/admin/http/handler.go

```diff
@@ -34,6 +39,11 @@ func getHandlers(bootstrap *bootstrapv1.Bootstrap, orchestrator *orchestrator.Or
 			"print cache entry for a given key. usage: `/cache/<key>`",
 			cacheDumpHandler(orchestrator),
 		},
+		{
+			"/log_level/",
+			"update the log level. usage: `/log_level/<level>`",
```

Comment:
May want to consider listing the options here: debug, info, warn, error, fatal

Reply:
Is it worth restricting the possible levels to be consistent with bootstrap.proto? What's the reasoning for restricting the possible levels?

Reply:
I just meant listing the levels specified in the proto here to help guide the endpoint user easier (without having to search up the proto spec).